### PR TITLE
Login event implementation

### PR DESCRIPTION
**Pull request by valentinBraeutigam**
@annikakrause

### CODE REVIEW

File: src/digitalIdentity-frontend/src/app/information-pop-up/information-pop-up.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<h2>{{header}}</h2>
+<p>{{text}}</p>
+
+<button class="btn btn-primary" (click)="close()">
```
Comment (Idontker) [inline src/digitalIdentity-frontend/src/app/information-pop-up/information-pop-up.component.html:4]: The button is missing the mat-button attribute.

File: src/digitalIdentity-frontend/src/app/information-pop-up/information-pop-up.component.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  selector: 'app-information-pop-up',
+  templateUrl: './information-pop-up.component.html',
+  styleUrls: ['./information-pop-up.component.css']
+})
+export class InformationPopUpComponent implements OnInit {
+  okayButtonString: string = "Okay"
+  header: string = ""
+  text: string = ""
+
+  constructor(private dialogRef: MatDialogRef<InformationPopUpComponent>,@Inject(MAT_DIALOG_DATA) private data:{header:string, text:string}) {
```
Comment (Idontker) [inline src/digitalIdentity-frontend/src/app/information-pop-up/information-pop-up.component.ts:15]: Great work on using an Dependency Injection/@Inject :D

**Review by Idontker — CHANGES_REQUESTED** [review]

File: src/digitalIdentity-frontend/src/app/app.module.ts [hunk 3]
```diff
@@ -18,7 +19,7 @@ import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MatNativeDateModule } from '@angular/material/core';
 import { HttpClientModule } from '@angular/common/http';
 import { MatTableModule } from '@angular/material/table'
-
+import { MatDialogModule } from '@angular/material/dialog'
```
[the remark this answers is not in the document]
Comment (valentinBraeutigam) [reply src/digitalIdentity-frontend/src/app/app.module.ts]: I moved it to the material.module. 
With moving them to specific components I would wait until we know if we don't need it also somewhere else.

File: src/digitalIdentity-frontend/src/app/information-pop-up/information-pop-up.component.html (hunk 1 above)
Comment (valentinBraeutigam) [reply src/digitalIdentity-frontend/src/app/information-pop-up/information-pop-up.component.html]: added :)

**Comment by valentinBraeutigam** [comment]
Thank you for the feedback :)

**Review by Idontker — APPROVED** [review]

**Pull request merged**